### DISCUSSION TITLE
Issue with using Chaste executable

### DISCUSSION THREAD

**Discussion by mythritg** [Chaste/Chaste#252] [Q&A] [answered]
I have been trying to use Chaste executable, but I am facing this error. Kindly guide me what to do here

![image](https://github.com/Chaste/Chaste/assets/166366098/93d00651-5c43-4545-a5eb-7c76408607b5)

**Comment by jmpf** [comment]
You are configuring the CMake build inside the apps subfolder (and pointing it at the apps subfolder).

I suggest:
```
mkdir ~/Chaste/build
cd ~/Chaste/build
cmake ..
```
Note the two dots on the last line is pointing cmake to the location of the Chaste main folder.

Hopefully that will configure correctly and you will see somewhere "Configuring Chaste app"

**Reply by mythritg** [reply]
Hi, 
No actually, the problem still pertains. I am trying to reinstall the entire software. But can you confirm if following [this ](https://chaste.github.io/docs/user-guides/cmake-first-run/) will build and configure the whole software including the executable?? 
Also, can you tell me what will be the path of chaste source code generally? Is it in my case ~/Chaste?

**Reply by jmpf** [reply]
Yes.  Those instructions will do for getting you configured.  They have more detail than https://chaste.github.io/docs/dev-guides/building-executable-apps/ (which I assume you were following).

In your case ~/Chaste is the path to Chaste.  (You ought to be able to type `cmake ~/Chaste` for the last line of my instructions above.

**Comment by jmpf** [comment, marked as answer]
> They have more detail than https://chaste.github.io/docs/dev-guides/building-executable-apps/ (which I assume you were following).

Once you are configured then do `make Chaste -j 4` to build (which will take time!).  Run with 
``` ./apps/Chaste ~/Chaste/apps/texttest/chaste/base_monodomain/ChasteParameters.xml ``` 
or similar.

**Reply by mythritg** [reply]
Thanks a lot, with this I was able to run the executable sample.
But, after configuration, https://chaste.github.io/docs/dev-guides/cmake-build-guide/ suggested testing, i.e. 
`ctest -L Continuous -j4`
and all the tests failed here for my case. Is something wrong with installation in my case??

**Reply by jmpf** [reply]
In what way are they failing?

Perhaps post the output of a small subset of the tests such as `ctest -L Continuous_io`

**Reply by mythritg** [reply]
![image](https://github.com/Chaste/Chaste/assets/166366098/10140c98-5dbc-4276-8f54-f04029efd650)
 This is the list of tests failed.

**Reply by jmpf** [reply]
I think you've unfortunately hit a known bug.  Your output looks identical to that reported by @ptheywood in #249.

Good news is that you have configured and compiled correctly!

**Reply by mythritg** [reply]
Yes, that is a relief! Thanks for your help
